File: Greencoin/GreenCoinMobile/lib/models/Address.dart
class Address {
  double latitude;
  double longitude;
  String areaOrStreet;
  String building;
  String flat;

  Address(
      {this.latitude,
      this.longitude,
      this.areaOrStreet,
      this.building,
      this.flat});

  Address.fromJson(Map<String, dynamic> json) {
    latitude = json['Latitude'].toDouble();
    longitude = json['Longitude'].toDouble();
    areaOrStreet = json['AreaOrStreet'];
    building = json['Building'];
    flat = json['Flat'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['Latitude'] = this.latitude;
    data['Longitude'] = this.longitude;
    data['AreaOrStreet'] = this.areaOrStreet;
    data['Building'] = this.building;
    data['Flat'] = this.flat;
    return data;
  }
}

File: Greencoin/GreenCoinService/GreenCoinService/GreenCoinService.DataContracts/Address.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GreenCoinService.DataContracts
{
    public class Address
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string AreaOrStreet { get; set; }
        public string Building { get; set; }
        public string Flat { get; set; }
    }
}
